### PR TITLE
[Feature] Monorepo Setup

### PR DESCRIPTION
**Pull request by nerdlet**
Add initial template for monorepo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,17 @@
-# ui
\ No newline at end of file
+# ui
+This project is using [pnpm](https://pnpm.io/) as a package manager.
```
Comment (kilemensi) [inline README.md:2]:
```suggestion
This project is using [pnpm](https://pnpm.io/) as a package manager.

```

File: apps/docs/.gitignore [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (kilemensi) [inline apps/docs/.gitignore:1]: A single .gitignore at root level should be enough

File: apps/docs/package.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev --port 3001",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "next": "latest",
```
Comment (kilemensi) [inline apps/docs/package.json:12]:
```suggestion
    "next": "^12.1.0",
```

File: apps/web/.gitignore [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```
Comment (kilemensi) [inline apps/web/.gitignore:1]: Not needed

File: apps/web/package.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "next": "latest",
```
Comment (kilemensi) [inline apps/web/package.json:12]:
```suggestion
    "next": "^12.1.0",
```

File: package.json [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  "name": "ui",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "build": "turbo run build",
+    "dev": "turbo run dev --parallel",
+    "lint": "turbo run lint"
+  },
+  "devDependencies": {
+    "eslint": "7.32.0",
```
Comment (kilemensi) [inline package.json:11]:
```suggestion
```

File: packages/config/package.json [hunk 7]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "config",
+  "version": "1.0.0",
```
Comment (kilemensi) [inline packages/config/package.json:3]:
```suggestion
  "version": "0.0.0",
```

File: packages/ui/package.json [hunk 8]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "ui",
+  "version": "0.0.0",
+  "main": "./index.js",
+  "license": "MIT",
+  "scripts": {
+    "lint": "eslint *.js"
+  },
```
Comment (kilemensi) [inline packages/ui/package.json:8]:
```suggestion
```

File: .eslintrc.js [hunk 9]
```diff
@@ -0,0 +1 @@
+module.exports = require("./packages/config/eslint-preset.js");
```
Comment (kilemensi) [inline .eslintrc.js:1]:
```suggestion
module.exports = require("./packages/config/eslint-preset.js");

```

**Review by kilemensi — COMMENTED** [review]
👍🏽

**Review by kilemensi — APPROVED** [review]
🚀

**Review by KobbyMmo — APPROVED** [review]
Looks good to me

**Pull request merged**